utils/src/integ-test/java/uk/gov/moj/sdt/test/utils/DBUnitUtility.java

```diff
@@ -1,635 +1,620 @@
+
+            // Generate the clean dataset, if loadRefData is true use
+            // clean_all.xml otherwise use clean.xml
+            final String cleanDataPath = "/clean/" + targetSchema + "_clean" + (reloadRefdata ? "_all" : "") + ".xml";
+            final IDataSet cleanDataset = builder.build(DBUnitUtility.class.getResourceAsStream(cleanDataPath));
+
+            // Disable triggers & constraints
+//            prepareForDbunitLoad(targetSchema);
+
+            // Clean the DB
+            LOGGER.debug("Cleaning database");
+            DatabaseOperation.TRUNCATE_TABLE.execute(dbConnection, cleanDataset);
+            dbConnection.getConnection().commit();
+
+            // If there's reference data insert it
+            // InputStream refDataStream =
+            // DBUnitUtility.class.getResourceAsStream("/refdata/" +
+            // targetSchema +
+            // "_refdata.xml");
+            // if (refDataStream != null && reloadRefdata)
+            // {
+            // LOGGER.debug("Inserting reference data");
+            // IDataSet referenceDataset = builder.build(refDataStream);
+            // DatabaseOperation.INSERT.execute(connection, referenceDataset);
+            // connection.getConnection().commit();
+            // }
+
+            // Enable triggers & constraints
+//            finishDbunitLoad(targetSchema);
+        } catch (final DatabaseUnitException e) {
+            LOGGER.error("Error loading data for Schema '" + targetSchema + "': " + e);
+            e.printStackTrace();
+        } catch (final IOException e) {
+            LOGGER.error("Error loading data for Schema '" + targetSchema + "': " + e);
+            e.printStackTrace();
+        } catch (final SQLException e) {
+            LOGGER.error("Error loading data for Schema '" + targetSchema + "': " + e);
+            e.printStackTrace();
+        }
+    }
+
+    /**
+     * Internal method to create a DBUnit connection to the DB defined in
+     * dbunit.properties.
+     *
+     * @param schema Target Schema
+     * @return DBUnit connection to database
+     */
+    private static IDatabaseConnection getConnectionInstance(final String schema) {
+        IDatabaseConnection dbConnection;
+
+        synchronized (allConnections) {
+            // Find the connection for this schema.
+            dbConnection = allConnections.get(schema);
+
+            // If no connection found for this schema, create one.
+            if (null == dbConnection) {
+                final Properties p = new Properties();
+                try {
+
+                    DataSource ds = SpringApplicationContext.getBean(DataSource.class);
+                    final Connection connection = ds.getConnection();
+                    dbConnection = new DatabaseConnection(connection, schema);
+                    dbConnection.getConfig().setProperty(DatabaseConfig.PROPERTY_DATATYPE_FACTORY,
+                            new PostgresqlDataTypeFactory());
```

Comment:
Previous version of this added an info message to the logger to say that a database connection had been created.  Might be worth including this again (possibly as a debug message).  The password can be excluded from the message if deemed necessary.